### ISSUE TITLE
Farmlist Animals check

### ISSUE THREAD

**Issue by 3kinex** [adipiciu/Travian-scripts#57]
**Script Name**
Travian Task Queue or Travian Wave Builder

**Server address**
https://cw.x1.international.travian.com

**User script manager**
Violentmonkey

**Operating System**
Windows 11

**Describe the bug**
Hi, so before update of farmlists, you were able to click on checkbox of farmlist to check them all and click on animasl with "???" then when in oasis were animals checkbox would dissappear. After update it doesn't dissappear, can you try to work it out?
Thanks for your hardwork. 

**To Reproduce**
1. Go to Farmlist
2. Click on Checkbox to check all targets, then on "Rat???" 
3.Checkboxes with animals won't dissappear

**Expected behavior**
That checkboxes that are marked and have animals in oasis would not be marked after checked for animals.

**Additional context**
![image_2023-12-27_203257700](https://github.com/adipiciu/Travian-scripts/assets/75512664/e20166b5-5ea5-4262-a623-6beceef32837)

**Comment by adipiciu** [comment]
Press F12 in browser, open the Console tab. Click on the rat to check for animals and then see if there is any error in the Console.

**Comment by FeaturedString** [comment]
Hi, not at home at this moment, i'll check that later. It woks like that on
opera gx.

**Comment by FeaturedString** [comment]
Not sure where i can see the errors, but its like that on opera gx.

**Comment by FeaturedString** [comment]
Doesnt show any errors beside "Deprecated Feature used" Also there's no "send all" button in  "Start all farm lists"

**Comment by adipiciu** [comment]
I don't have an account on a server with start all farm lists option enabled.

**Comment by FeaturedString** [comment]
It's available on international servers

**Comment by adipiciu** [comment]
Ok, but it takes many days to make an account and enable farm lists. I will try these days.

**Comment by adipiciu** [comment]
> It's available on international servers

It seems it's on all servers :)

**Comment by adipiciu** [comment]
The checkboxes have a very strange behavior. 
If you manually select an oasis, click the Rat to check the oasis -> the script will uncheck the oasis.
If you select all farms from the game button, click the Rat to check the oasis -> script will uncheck the oasis, but the game will check back the oasis because of the All Farms button is selected.

I will need to think how to fix this.

**Comment by adipiciu** [comment]
> Travian Task Queue or Travian Wave Builde

Yes. This is in the other script. I will update that later. I need to fix more things in it.

**Comment by FeaturedString** [comment]
okay, got you. Will be waiting for it. Also is there a plan for what features are you planning to add to script? Would be nice to read through and see what we can use in future.

**Comment by adipiciu** [comment]
I have no plans to add new features as I am no longer actively playing. I am only doing bugfixes mostly.

> okay, got you. Will be waiting for it. Also is there a plan for what features are you planning to add to script? Would be nice to read through and see what we can use in future.

**Comment by FeaturedString** [comment]
> ![image](https://private-user-images.githubusercontent.com/75512664/293370067-e8af185e-1bca-4e8c-b325-59ee1f84216a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwODA2NDksIm5iZiI6MTcwNTA4MDM0OSwicGF0aCI6Ii83NTUxMjY2NC8yOTMzNzAwNjctZThhZjE4NWUtMWJjYS00ZThjLWIzMjUtNTllZTFmODQyMTZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE3MjU0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjYTIxNWU0N2U5M2I0YThiODNjMTY1OTE1ZWI2ZmUzMzUzMjRiZWY2M2Q3MTJjMTNiOWFkN2ZmOGE1ZDg3MzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZqhU00eB7CNuM5IWBKLsX0GLYb_PfIUld-9MiEDkT70) this is what i meant, theres no send later for this button

is this still being worked on?

**Comment by adipiciu** [comment]
> is this still being worked on?

It's on my list, but I'm not currently working on it. No estimated time yet.

**Comment by adipiciu** [comment]
> is this still being worked on?

Added in TTQ 2.0.12
